client/public/index.html

```diff
@@ -3,6 +3,8 @@
   <head>
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <link href="https://fonts.googleapis.com/css?family=Roboto:300,400,500" rel="stylesheet">
+    <link href="%PUBLIC_URL%/css/styles.css" rel="stylesheet">
     <title>EviteHub</title>
```

Comment:
Same as above. We don't need to call style for font and custom styles in the HTML